CTC/src/ctc/utils/Clock.java

```diff
@@ -0,0 +1,72 @@
+package ctc.utils;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+
+public class Clock implements ClockInterface {
+
+  private static Calendar calendar;
+  private static DateFormat dateFormat =
+      new SimpleDateFormat("HH:mm:ss");
+  private static long start;
```

Comment:
Will you ever set start and end? If no, one of them should be a function variable and the remaining global could be changed to lastTimestamp.